Gemfile.lock

```diff
@@ -55,4 +55,4 @@ DEPENDENCIES
   vcr (~> 4.0.0)
 
 BUNDLED WITH
-   1.16.2
+   2.0.1
```

Comment:
Seems like bumping this is making CircleCI break. Either don't bump this as part of this PR or fix CircleCI :P, although if this is going be a major release (7.0) it might be a good time to do these sorts of updates.

Reply:
I don't actually have access to circle CI so I'm just going to revert the bundler upgrade for now